lib/axp209.js

```diff
@@ -15,12 +15,16 @@ function AXP209(bus, address) {
   this._i2c = new I2C(bus, address);
 
   this._reads = {};
+  
 }
 
 util.inherits(AXP209, events.EventEmitter);
 
 AXP209.prototype.open = function() {
   this._i2c.open();
+  
+  this._configureVcc18Adc();
+  //this._resetVcc18();
```

Comment:
Close, just remember mode is a number see: https://github.com/achingbrain/board-io/blob/master/lib/constants.js#L7